application/pipeline-stacks/oncoanalyser/Dockerfile

```diff
@@ -40,11 +41,16 @@ RUN \
   apt update -y -q && \
   apt install -y -q \
     curl \
-    jq="${JQ_VERSION}*" \
```

Comment:
What is the reason for switching to a manual install for jq?

Reply:
Debian 11 installs jq v1.6 by default but we need v1.7 for the following:

```
      .inputs.mode as $mode |
      if any($valid_modes_array[] == $mode) then
        $mode
      else
        null
      end
```

Reply:
Got it, I'll move the jq install to Conda outside this PR after merging so we don't have to rely on manual installation